src/ui/server/helpers/mappings/map-ordnance-survey-addresses/is-checked/index.ts

```diff
@@ -0,0 +1,25 @@
+import { MappedOrdnanceSurveyAddress, ApplicationBroker } from '../../../../../types';
+
+/**
+ * isChecked
+ * Check if a provided address is a submitted broker address.
+ * @param {MappedOrdnanceSurveyAddress} Ordnance survey address
+ * @param {ApplicationBroker} Application broker address
+ * @returns {Boolean}
+ */
+const isChecked = (address: MappedOrdnanceSurveyAddress, submittedAddress: ApplicationBroker) => {
+  const { addressLine1, addressLine2, town, postcode } = address;
+
+  if (
+    addressLine1 === submittedAddress.addressLine1 &&
```

Comment:
can this be a const please?
Can this be a ternary operator please?

Reply:
No need for a ternary operator with consts